tests/core/ingestion/helpers.py

```diff
@@ -78,8 +78,9 @@
 SLUG_DOCUMENT_NAME = "DocSlug1"
 COLLECTION_IMPORT_ID = "CPR.Collection.1"
 
-FOUR_EVENT_ROWS = """Id,Eventable type,Eventable Id,Eventable name,Event type,Title,Description,Date,Url,CPR Event ID,CPR Family ID,Event Status
+FIVE_EVENT_ROWS = """Id,Eventable type,Eventable Id,Eventable name,Event type,Title,Description,Date,Url,CPR Event ID,CPR Family ID,Event Status
 1101,Legislation,1001,Title1,Passed/Approved,Published,,2019-12-25,,CCLW.legislation_event.1101.0,CCLW.family.1001.0,OK
+1101,Legislation,1001,Title1,Amended,Amended,,2021-12-25,,CCLW.legislation_event.1101.1,CCLW.family.1001.0,OK
```

Comment:
Do we also want to test that an additional and later change to title is correctly used?

Reply:
have modified one of the tests to show last updated date changing as events are added.